<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1582</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widgetBase" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="Background" stdset="0">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QWidget" name="widgetTitle" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(254, 254, 254);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="rightMargin">
           <number>24</number>
          </property>
          <property name="bottomMargin">
           <number>18</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lbLogo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/Resourse/image/logo.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbTitle">
              <property name="text">
               <string>深海研究所</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>741</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>48</number>
            </property>
            <item>
             <widget class="Button" name="btnMin">
              <property name="text">
               <string/>
              </property>
              <property name="MenuButton" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Button" name="btnMax">
              <property name="text">
               <string/>
              </property>
              <property name="MenuButton" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Button" name="btnClose">
              <property name="text">
               <string/>
              </property>
              <property name="MenuButton" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widgetStart" native="true">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>237</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>186</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnOpen">
            <property name="text">
             <string>浏览文件</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>186</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>237</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widgetContent" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="Card" name="widgetContainer1" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>211</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QtAvPlayer" name="widgetPlayer0" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>4K 高清</string>
             </property>
             <property name="VideoTitle" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widgetPlayerList" native="true">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>70</y>
            <width>451</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayoutPlayerList">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <property name="spacing">
            <number>9</number>
           </property>
           <item row="3" column="0">
            <widget class="Card" name="widgetContainer5" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="1" column="0">
               <widget class="QtAvPlayer" name="widgetPlayer4" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>云台照相</string>
                </property>
                <property name="VideoTitle" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="Card" name="widgetContainer4" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="1" column="0">
               <widget class="QtAvPlayer" name="widgetPlayer3" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>右侧内置高清</string>
                </property>
                <property name="VideoTitle" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="Card" name="widgetContainer8" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="1" column="0">
               <widget class="QtAvPlayer" name="widgetPlayerInfo" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true">color: rgb(163, 163, 163);
font: 8pt &quot;微软雅黑&quot;;</string>
                </property>
                <property name="text">
                 <string>实时位置</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="Card" name="widgetContainer6" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="1" column="0">
               <widget class="QtAvPlayer" name="widgetPlayer5" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>正前广角高清</string>
                </property>
                <property name="VideoTitle" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="Card" name="widgetContainer7" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="1" column="0">
               <widget class="QtAvPlayer" name="widgetPlayer6" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>左内置高清</string>
                </property>
                <property name="VideoTitle" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="Card" name="widgetContainer3" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="1" column="0">
               <widget class="QtAvPlayer" name="widgetPlayer2" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>艉部高清</string>
                </property>
                <property name="VideoTitle" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1" rowspan="4">
            <widget class="Card" name="widgetDynamicInfo" native="true">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QWidget" name="widget" native="true"/>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btnOpenVideos">
                <property name="text">
                 <string>选择视频</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="Card" name="widgetContainer2" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="1" column="0">
               <widget class="QtAvPlayer" name="widgetPlayer1" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>底部高清</string>
                </property>
                <property name="VideoTitle" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="WidgetMediaControl" name="widgetMediaControl" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>480</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>Public/button.h</header>
  </customwidget>
  <customwidget>
   <class>QtAvPlayer</class>
   <extends>QWidget</extends>
   <header>Player/qtavplayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Card</class>
   <extends>QWidget</extends>
   <header>Public/card.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetMediaControl</class>
   <extends>QWidget</extends>
   <header>Public/widgetmediacontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
